frontend/src/constants/zipCodesByCountyByState.ts

```diff
@@ -1328,6 +1328,1888 @@ export const ZIPS_BY_COUNTY_BY_STATE: Record<
       '35650'
     ]
   },
```

Comment:
If we're not going to use the full service areas for filtering, we could considering stopping this map at the county level. Thoughts @viviandien @tetraptych ?

Reply:
While we don't need them for public launch, we may want them eventually since a lot of health care plans consider service area by ZIP. As long as is it's easy to access ZIPs when we want them in the future, I'm ok with whatever.

Reply:
But do we do an AND or an OR? Philosophy essay, you have 1 hour.

Reply:
I like the idea though